test/issue895-local-configuration.sh

```diff
@@ -14,9 +15,11 @@ if [ -e ~/.dub/settings.json ]; then
 	die $LINENO 'Found existing user wide DUB configuration. Aborting.'
 fi
 
-if ! { ${DUB} describe --single issue103-single-file-package.d 2>&1 || true; } | grep -cF 'Unknown compiler: foo'; then
+if ! { ${DUB} describe --single issue103-single-file-package.d 2>&1 || true; } | grep -cF 'Unknown compiler: '; then
```

Comment:
This test should now pass, but removed the compiler name `foo` because my new code passes the full path of a compiler next to `dub` to the settings and this error output.  I'm adding more tests for the new behavior and will update the patch soon.